src/App.tsx

```diff
@@ -8,7 +8,28 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [status, setStatus] = useState('all');
+
+  useEffect(() => {
+    getTodos().then(result => setTodos(result));
```

Comment:
```suggestion
    getTodos().then(response => setTodos(response));
```
or even better `todosFromServer`
use meaningful names of variables